tests/components/uptime/test_sensor.py

```diff
@@ -1,90 +1,85 @@
-        """Test uptime sensor with minutes defined in config."""
-        config = {"sensor": {"platform": "uptime", "unit_of_measurement": "minutes"}}
-        assert setup_component(self.hass, "sensor", config)
-
-    def test_uptime_sensor_days_output(self):
-        """Test uptime sensor output data."""
-        sensor = UptimeSensor("test", "days")
-        assert sensor.unit_of_measurement == "days"
-        new_time = sensor.initial + timedelta(days=1)
-        with patch("homeassistant.util.dt.now", return_value=new_time):
-            asyncio.run_coroutine_threadsafe(
-                sensor.async_update(), self.hass.loop
-            ).result()
-            assert sensor.state == 1.00
-        new_time = sensor.initial + timedelta(days=111.499)
-        with patch("homeassistant.util.dt.now", return_value=new_time):
-            asyncio.run_coroutine_threadsafe(
-                sensor.async_update(), self.hass.loop
-            ).result()
-            assert sensor.state == 111.50
-
-    def test_uptime_sensor_hours_output(self):
-        """Test uptime sensor output data."""
-        sensor = UptimeSensor("test", "hours")
-        assert sensor.unit_of_measurement == "hours"
-        new_time = sensor.initial + timedelta(hours=16)
-        with patch("homeassistant.util.dt.now", return_value=new_time):
-            asyncio.run_coroutine_threadsafe(
-                sensor.async_update(), self.hass.loop
-            ).result()
-            assert sensor.state == 16.00
-        new_time = sensor.initial + timedelta(hours=72.499)
-        with patch("homeassistant.util.dt.now", return_value=new_time):
-            asyncio.run_coroutine_threadsafe(
-                sensor.async_update(), self.hass.loop
-            ).result()
-            assert sensor.state == 72.50
-
-    def test_uptime_sensor_minutes_output(self):
-        """Test uptime sensor output data."""
-        sensor = UptimeSensor("test", "minutes")
-        assert sensor.unit_of_measurement == "minutes"
-        new_time = sensor.initial + timedelta(minutes=16)
-        with patch("homeassistant.util.dt.now", return_value=new_time):
-            asyncio.run_coroutine_threadsafe(
-                sensor.async_update(), self.hass.loop
-            ).result()
-            assert sensor.state == 16.00
-        new_time = sensor.initial + timedelta(minutes=12.499)
-        with patch("homeassistant.util.dt.now", return_value=new_time):
-            asyncio.run_coroutine_threadsafe(
-                sensor.async_update(), self.hass.loop
-            ).result()
-            assert sensor.state == 12.50
+
+
+async def test_uptime_min_config(hass):
+    """Test minimum uptime configuration."""
+    config = {"sensor": {"platform": "uptime"}}
+    assert await async_setup_component(hass, "sensor", config)
+    await hass.async_block_till_done()
+    state = hass.states.get("sensor.uptime")
+    assert state.attributes.get("unit_of_measurement") == "days"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)